frontend/src/components/pages/logHours.tsx

```diff
@@ -368,23 +379,15 @@ export default function LogHours({
 
           <StyledLabel htmlFor="notes">Additional Notes</StyledLabel>
           {thisEvent ? (
-            <textarea
-              rows={5}
-              cols={50}
-              style={{
-                fontSize: 20,
-                borderRadius: 10,
-                resize: "none",
-                fontFamily: "Poppins",
-                border: "1px solid #c4c4c4",
-                paddingLeft: 10,
-                margin: "5px 0 20px 0",
-                width: "100%",
-              }}
-              value={notes}
-              onChange={(e) => setNotes(e.target.value)}
-              disabled={new Date(thisEvent.start) > new Date()}
-            />
+            <TextArea>
+              <textarea
+                rows={5}
+                cols={50}
+                value={notes}
+                onChange={(e) => setNotes(e.target.value)}
+                disabled={new Date(thisEvent.start) > new Date()}
+              />
```

Comment:
Once you change the styled component to a textarea, you can apply these properties directly to the styled component rather than nesting another textarea